test/Libraries/RevitIntegrationTests/RegressionTests.cs

```diff
@@ -159,7 +159,7 @@ public void NoDuplicatePortsOnUINodes()
             Assert.NotNull(homespace, "The current workspace is not a HomeWorkspaceModel");
 
             // Iterate through all loaded nodes in library & add Revit UI nodes to ws
-            var nodeList = Model.CurrentWorkspace.Nodes.OfType<NodeSearchElement>();
+            var nodeList = Model.SearchModel.Search(string.Empty, Model.LuceneUtility);
```

Comment:
this returns all nodes in search?

Reply:
It did not, I fixed it, but I still have other issues with these tests. I'll commit the current state, but it's not yet ready.